configs/mmdet/mmdet_regression_test.yaml

```diff
@@ -0,0 +1,182 @@
+  dataset_path:
+    COCO:
+  metric_tolerance:
+    box AP: 0.1
+    mask AP: 0.1
+    PQ: 0.1
+
+default_config: &default_config
+  input_img_path: &default_input_img_path ./tests/data/tiger.jpeg
+  dynamic_test_img_path: &default_dynamic_test_img_path
+  sdk_config: &default_sdk_config
+    deploy_config:
+      static:
+        fp32: &default_sdk_static_fp32
+      dynamic:
+        fp32: &default_sdk_dynamic_fp32 detection_sdk_dynamic.py
+  backends:
+    onnxruntime: &default_onnxruntime_config
+      performance_align: False
+      dynamic_test_img_path:
+      deploy_config:
+        static:
+          fp32: &default_onnxruntime_static_fp32 detection_onnxruntime_static.py
+        dynamic:
+          fp32: &default_onnxruntime_dynamic_fp32 detection_onnxruntime_dynamic.py
+    tensorrt: &default_tensorrt_config
+      performance_align: False
+      dynamic_test_img_path:
+      deploy_config:
+        static:
+          fp32: &default_tensorrt_static_fp32 detection_tensorrt_static-800x1344.py
+          fp16: &default_tensorrt_static_fp16 detection_tensorrt-fp16_static-800x1344.py
+          int8: &default_tensorrt_static_int8 detection_tensorrt-int8_static-800x1344.py
+        dynamic:
+          fp32: &default_tensorrt_dynamic_fp32 detection_tensorrt_dynamic-320x320-1344x1344.py
+          fp16: &default_tensorrt_dynamic_fp16 detection_tensorrt-fp16_dynamic-320x320-1344x1344.py
+          int8: &default_tensorrt_dynamic_int8 detection_tensorrt-int8_dynamic-320x320-1344x1344.py
+    openvino: &default_openvino_config
+      performance_align: False
+      dynamic_test_img_path:
+      deploy_config:
+        static:
+          fp32: &default_openvino_static_fp32
+        dynamic:
+          fp32: &default_openvino_dynamic_fp32 detection_openvino_dynamic-800x1344.py
+    ncnn: &default_ncnn_config
+      performance_align: False
+      dynamic_test_img_path:
+      deploy_config:
+        static:
+          fp32: &default_ncnn_static_fp32 single-stage_ncnn_static-800x1344.py
+        dynamic:
+          fp32: &default_ncnn_dynamic_fp32 single-stage_ncnn_dynamic.py
+    pplnn: &default_pplnn_config
+      performance_align: False
+      dynamic_test_img_path:
+      deploy_config:
+        static:
+          fp32: &default_pplnn_static_fp32
+        dynamic:
+          fp32: &default_pplnn_dynamic_fp32 detection_pplnn_dynamic-800x1344.py
+    torchscript: &default_ts_config
+      performace_align: False
+      dynamic_test_img_path:
+      deploy_config:
+        dynamic:
+          fp32: &default_ts_dynamic_fp32 detection_torchscript.py
+
+models:
+  - name: retinanet
+    codebase_model_config_dir: ./configs/retinanet
+    model_configs:
+      - retinanet_r50_fpn_1x_coco.py
+    metafile: configs/retinanet/metafile.yml
+    deploy_config_dir: ./configs/mmdet/detection
+    input_img_path: *default_input_img_path
+    dynamic_test_img_path:
+    calib_dataset_cfg:
+    sdk_config:
+      deploy_config_dir: ./configs/mmdet/detection
```

Comment:
deploy_config_dir should be moved to global_info

Reply:
Done